packages/app/features/home/screen.tsx

```diff
@@ -1,41 +1,231 @@
-// import { createPasskey, signWithPasskey } from '@daimo/expo-passkeys'
-import { Button, Container, H1, YStack } from '@my/ui'
+import { Anchor, Avatar, Button, Card, Container, H1, Paragraph, XStack, YStack, Dialog, ScrollView, ListItem, Theme } from '@my/ui'
+import { useThemeSetting } from '@tamagui/next-theme'
+import { Image } from '@my/ui'
+import { MainLayout } from '../../components/layout';
```

Comment:
us the tsconfig paths setting.

```suggestion
import { MainLayout } from 'app/components/layout';
```